provider/scalewayProvider.js

```diff
@@ -38,22 +38,36 @@ class ScalewayProvider {
   }
 
   setCredentials(options) {
+    // On serverless info command we do not want log pollution from authentication.
+    // This is necessary to use it in automated environment.
+    let hideLog = false;
+    if (this.serverless.configurationInput.service && this.serverless.configurationInput.service === 'serverlessInfo') {
+      hideLog = true;
+    }
     if (options['scw-token'] && options['scw-project']) {
-      this.serverless.cli.log('Using credentials from command line parameters');
+      if (!hideLog) {
```

Comment:
This is ok for now, but eventually we should use the latest Serverless Framework logging API as shown here: https://www.serverless.com/framework/docs/guides/plugins/cli-output

We would put information like the current `Using credentials...` as debug statements and have the default log level at info.

I've linked this in our ticket to refactor our plugin code.